README.md

```diff
@@ -14,4 +14,6 @@ This is a collaboration library for solving time travel.
 # Warnings
 
 Time travel can create time paradoxes such as the grandfather paradox and the bootstrap paradox.
-Proceed with care.
\ No newline at end of file
+Proceed with care.
+
+and take care not to push directly to main
```

Comment:
I would like the reason explained for why we should not push directly to the main.